Ge09 team6

lare-odoo opened the pull request.

cogs-odoo committed:
[IMP] Improvement on file format

Improved the format of some files and delete unnecessary imports and files

Commit:
[IMP] GE09-TEAM6 : Add  motorcycle specific info to the repair view and create a smart button that makes a relation between the montorcyclke registry to the repair order

We added a vin field to the repair view to compute specific values related to it, after that we make a relationship between motorcycle registry and repair order, compute a variable
that counts how many repair orders there are then we created a function to show the tree view of the repair orders with a domain to only show the repair orders that we expect

Commit:
[IMP] GE09-TEAM6 : Add  motorcycle specific info to the repair view and create a smart button that makes a relation between the montorcyclke registry to the repair order

We added a vin field to the repair view to compute specific values related to it, after that we make a relationship between motorcycle registry and repair order, compute a variable
that counts how many repair orders there are then we created a function to show the tree view of the repair orders with a domain to only show the repair orders that we expect

[IMP] GE09-TEAM6: 'lot_id' is now automatically obtained from motorcycle registry.

This change facilitates the automation of processes like creating a repair through a portal.

[IMP] GE09-TEAM6: Resolved formatting issues

Changed double quotes to single quotes to maintain consistency between files. Also, removed trailing whitespace from line breaks at the end of some files

lare-odoo committed:
[IMP] GE09-TEAM6: 'lot_id' is now automatically obtained from motorcycle registry.

This change facilitates the automation of processes like creating a repair through a portal.

yall-odoo commented on GE09-TEAM6/models/repair_order.py:
```diff
@@ -9,6 +9,8 @@ class RepairOrder(models.Model):
     sale_order_id = fields.Many2one(related='registry_id.sale_order_id')
     product_id = fields.Many2one(related='registry_id.lot_id.product_id')
 
+    lot_id = fields.Many2one(related="registry_id.lot_id")
```
Commit to using double quotes or single quotes, not both.

lare-odoo committed:
[IMP] GE09-TEAM6: Resolved formatting issues

Changed double quotes to single quotes to maintain consistency between files. Also, removed trailing whitespace from line breaks at the end of some files

lare-odoo committed:
GE09-TEAM6: Merge branch 'GE09-TEAM6' of github.com:cogs-odoo/team6-kawiil-motors into GE09-TEAM6

cogs-odoo committed:
[IMP] Improvement on file format

cogs-odoo committed:
[MERGE] conflicts between 11 and 09

cogs-odoo committed:
[IMP] Improvement on coding guideline
